src/pages/MyProfile/MyProfile.js

```diff
@@ -3,39 +3,46 @@ import TableItem from '../../Components/Table-Item/TableItem';
 import './MyProfile.css';
 
 const MyProfile = () => {
-  const missions = [
-    {
-      name: 'Russia',
-      id: '1',
-    },
-    {
-      name: 'Norway',
-      id: '2',
-    },
-    {
-      name: 'Canada',
-      id: '3',
-    },
-    {
-      name: 'Mexico',
-      id: '4',
-    },
-    {
-      name: 'Cuba',
-      id: '5',
-    },
-  ];
+  // const missions = [
+  //   {
+  //     name: 'Russia',
+  //     id: '1',
+  //   },
+  //   {
+  //     name: 'Norway',
+  //     id: '2',
+  //   },
+  //   {
+  //     name: 'Canada',
+  //     id: '3',
+  //   },
+  //   {
+  //     name: 'Mexico',
+  //     id: '4',
+  //   },
+  //   {
+  //     name: 'Cuba',
+  //     id: '5',
+  //   },
+  // ];
```

Comment:
Please remove these comments

Reply:
Thanks for your review, changes will be implemented